tests/common.go

```diff
@@ -3478,6 +3478,50 @@ func CreateSourceAndDestClusters(orgID string, cloudName string, uid string, ctx
 	return nil
 }
 
+// RegisterCluster register the cluster in px-backup based on the provided cluster name
+func RegisterCluster(orgID string, clusterName string, cloudName string, uid string, ctx context1.Context) error {
```

Comment:
Few comments about this function:

- Move this function to `backup_helper.go`
- Rename `cloudName` to `cloudCredName`
- UID is not a mandatory parameter, so we dont have to pass it. And even in future if we have to pass it, we can generate it inside the function rather than making the caller pass it
- nit: Doc string should say `RegisterCluster adds the cluster with the given name`
- In my opinion, we can remove this function and just use `CreateCluster` function by writing a couple of wrapper functions like `AddSourceCluster(clusterName string, ctx context1.Context) error`
 and `AddDestinationCluster(clusterName string, ctx context1.Context) error`. These two functions will internally derive the kubeconfig path and pass it on to `CreateCluster` function

- In future, if we need to add a third cluster, we can ask the caller to directly call the `CreateCluster` function

Reply:
I added this function keeping in mind  addition of new clusters.
If can can write a function GetClusterConfigPath which can be called inside RegisterCluster  to get the kubeconfig 
based on the key/parameter passed from test ..

Please suggest